### DISCUSSION TITLE
BLOM tag for NorESM2.3 release

### DISCUSSION THREAD

**Discussion by TomasTorsvik** [NorESMhub/BLOM#339] [General]
@matsbn , @jmaerz , @JorgSchwinger , @milicak 
We should soon make a tag on BLOM for the upcoming NorESM2.3 release. I suggest we tag with `v1.6.0`, unless there is a reason for  changing the major version to `v2.0.0`. Does anyone have an opinion what would trigger a new major version, or do we just continue incrementing the minor release number?

**Comment by JorgSchwinger** [comment]
Making hybrid coordinates the default would be a reason for v2.0.0, but we still don't do this?

**Reply by TomasTorsvik** [reply]
I agree that hybrid coordinates as default is a candidate for v2.0.0. We are not planning to make do this for NorESM2.3, but it is open for discussion for NorESM2.5.
